### ISSUE TITLE
Allow saving all labels from a repository as a package

### ISSUE THREAD

**Issue by Jameskmonger** [himynameisdave/git-labelmaker#46]
I think we should allow users to dump all the labels from a repository, so that they can then apply that setup to other repositories.

This is blocked by #43

**Comment by Jameskmonger** [comment]
@himynameisdave I think that 0.8.0 is the best milestone for this as it is new functionality.

**Comment by TheCodeDestroyer** [comment]
:+1: for this

**Comment by himynameisdave** [comment]
:+1: this is a great idea

**Comment by NewAlexandria** [comment]
#43 was closed, and another PR will be opened that also precedes this one (#46)?  

I don't mind either way, and cheers + thanks to you guys for adding the functionality.  Just following along here.

**Comment by Jameskmonger** [comment]
The issue has been fixed in 0.7.3 which will be released soon.

@himynameisdave is there any chance you could release a 0.7.3-alpha for @NewAlexandria to use?

**Comment by himynameisdave** [comment]
@Jameskmonger Yes this is a good idea, gonna merge your #50 first then I'll release that. We'll also deprecate the old versions like in #47

**Comment by NewAlexandria** [comment]
:+1:  look forward to it.
